### PR TITLE
Added cc audit logging for service starts

### PR DESCRIPTION
**Pull request by svambati**
```
INFO[0274] Starting Service                              action=go id=b4bsouhkqvb1z04g7qc4898xw success=true type=service user=sambati
INFO[0274] Service Scheduled                             action=go id=doygznlb97zxxhwbrgmk8219v servicename=HMaster success=true type=service user=system
INFO[0274] Service Scheduled                             action=go id=aye1jymgafw3eknb9p5jkdyj8 servicename=redis success=true type=service user=system
INFO[0274] Service Scheduled                             action=go id=1lfc4i5q5iot2a4iz3igjo0bm servicename=mariadb-events success=true type=service user=system
INFO[0274] Service Scheduled                             action=go id=6xf9sz2xblo2qbg78iit1qeni servicename=ZooKeeper success=true type=service user=system
INFO[0274] Service Scheduled                             action=go id=2rag50zn0cmbks4zeyc51xu6s servicename=memcached success=true type=service user=system
INFO[0274] Service Scheduled                             action=go id=aczzbfxahilxrl8dypnw1dps servicename=mariadb-model success=true type=service user=system
INFO[0274] Service Scheduled                             action=go id=3m4i3w3vpharnfhxnuocdn92a servicename=RabbitMQ success=true type=service user=system
INFO[0274] Service Scheduled                             action=go id=6vl9ppo83gk5sgmu1tz5mip30 servicename=Solr success=true type=service user=system
```

### CODE REVIEW

**Comment by svambati** [comment]
retest this please

File: facade/service.go [hunk 1]
```diff
@@ -1788,7 +1800,9 @@ func (f *Facade) WaitSingleService(svc *service.Service, dstate service.DesiredS
 
 func (f *Facade) StartService(ctx datastore.Context, request dao.ScheduleServiceRequest) (int, error) {
 	defer ctx.Metrics().Stop(ctx.Metrics().Start("Facade.StartService"))
-	return f.ScheduleServices(ctx, request.ServiceIDs, request.AutoLaunch, request.Synchronous, service.SVCRun, false)
+	alog := f.auditLogger.Action(audit.Start).Message(ctx, "Starting Service").Type(service.GetType()).ID(strings.Join(request.ServiceIDs, ", "))
```
Comment (tgjonestx) [inline facade/service.go]: A couple of suggestions to emphasize that the user is starting one or more services:
* `"Starting Service(s)"`
* `WithField("ids", strings.Join(request.ServiceIDs, ", "))` instead if `ID(...)`
* Also add a field for `count` and log successCount

Comment (tgjonestx) [inline facade/service.go]: A couple of suggestions to emphasize that the user is starting one or more services:
* `"Starting Service(s)"`
* `WithField("ids", strings.Join(request.ServiceIDs, ", "))` instead if `ID(...)`
* Also add a field for `count` and log successCount

File: audit/action.go [hunk 2]
```diff
@@ -29,11 +30,32 @@ const (
...
+	case "go":
+		return Start
+	case "stop":
+		return Stop
+	case "pause":
+		return Pause
+	default:
+		return "unknown"
+	}
+}
```
Comment (tgjonestx) [inline audit/action.go]: Please move the maping to DesiredState - something like this:
```
func (state DesiredState) ToAuditAction() string {
	switch state {
	case SVCRestart:
		return "restart"
	case SVCStop:
		return "stop"
	case SVCRun:
		return "start"
	case SVCPause:
		return "pause"
	default:
		return "unknown"
	}
}
```

**Pull request merged**